package.json

```diff
@@ -87,7 +92,9 @@
     "@aws-cdk/aws-logs": "1.x",
     "@aws-cdk/aws-sns": "1.x",
     "@aws-cdk/aws-sns-subscriptions": "1.x",
-    "@aws-cdk/core": "1.x"
+    "@aws-cdk/core": "1.x",
```

Comment:
We should maybe now mark that we need 1.75.0 or greater version for cdk.

Similar minimum requirement should maybe be added to esbuild

Reply:
good point, current setup is too vague

Reply:
got me thinking, @aws-cdk/aws-lambda-nodejs is in experimental state so we should be quite strict with its version.